civihr_default_theme/scss/components/_panel.scss

```diff
@@ -244,14 +237,12 @@ form {
 }
 
 .chr_panel--my-details__data-group {
-
   > .chr_panel--my-details__data-group__title {
     margin-bottom: 20px;
   }
 }
 
-.chr_panel--my-details__data-group__content {
-
+.chr-panel--my-details--view--panel {
```

Comment:
Naming is wrong, as we need to follow BEM, `__` and `--` is not the same. It should be `chr_panel--my-details__view__panel`.

Reply:
Views does not allow me to put a class with underscores, it automatically transforms the class name (see the exported view which has the correct class name).

Reply:
See here, this is a bug in views https://www.drupal.org/project/views/issues/1371118#comment-12228781

Reply:
@deb1990 in our last conversation I told you that I will change views templates for _My Details_ page to output this class correctly to be able to comply with BEM formatting.